spec/features/profile_spec.rb

```diff
@@ -13,11 +13,11 @@
       visit profile_account_path
     end
 
-    it { page.should have_content("Remove account") }
+    it { expect(page).to have_content("Remove account") }
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.